closed/OpenJ9.gmk

```diff
@@ -200,8 +200,8 @@ stage-j9 : \
 
 # Only update version files when the SHAs change.
 $(OUTPUT_ROOT)/vm/compiler/jit.version : $(call DependOnVariable, OPENJ9_SHA)
-        @$(MKDIR) -p $(@D)
-        $(ECHO) '#define TR_LEVEL_NAME "$(OPENJ9_SHA)"' > $@
+  @$(MKDIR) -p $(@D)
```

Comment:
Should this be a tab?  The version for the old tr.source/jit.version file uses tab

Reply:
Agreed, it appears to be 2 spaces which is still going to be a problem.

Reply:
@DanHeidinga updated PTAL